vuepress/docs/next/tutorials/create/ms/run-local.md

```diff
@@ -66,19 +66,18 @@ ent bundle run conference-details
 ```
 When the run is complete, you should see the details MFE with the ID 1 loaded.
 
-<!-- >>If you want to modify a different row in the database, edit `microfrontends/conference-details/public/index.html` file. Change the `id` attribute in this line:
-
-    <my-entity-details service-url="%REACT_APP_SERVICE_URL%" id="1" />
--->
+>If you want to modify a different row in the database, edit `microfrontends/conference-details/public/index.html` file. Change the `id` attribute in this line:
+``` html
+    <my-entity-details id="1" />
```

Comment:
I know I'm forecasting a bit but let's match the test usecase, so this should be `<conference-details id="1"/>`